packages/accordions/package.json

```diff
@@ -33,8 +33,10 @@
     "styled-components": "^4.2.0"
   },
   "devDependencies": {
+    "@types/lodash.debounce": "^4.0.6",
```

Comment:
Should the `lodash.debounce` dependencies be pinned?

Reply:
Oops, I'll go ahead and update this, thanks for catching this!